src/test/unit/wrappers/http/test_http_parser.py

```diff
@@ -122,18 +127,191 @@ def test_lambda_parser_resource_name(uri, resource_name):
         b'<?xml version="1.0"?><SendMessageResponse xmlns="http://queue.amazonAwsParser.com/doc/2012-11-05/"><SendMessageResult><MessageId>85dc3997-b060-47bc-9d89-c754d7260dbd</MessageId><MD5OfMessageBody>c5cb6abef11b88049177473a73ed662f</MD5OfMessageBody></SendMessageResult><ResponseMetadata><RequestId>b6b5a045-23c6-5e3a-a54f-f7dd99f7b379</RequestId></ResponseMetadata></SendMessageResponse>',
     ],
 )
-def test_sqs_parser_message_id(body):
-    response = SqsParser().parse_response("dummy", 200, {}, body=body)
+def test_sqs_xml_parser_message_id(body):
+    response = SqsXmlParser().parse_response("dummy", 200, {}, body=body)
     assert response["info"]["messageId"] == "85dc3997-b060-47bc-9d89-c754d7260dbd"
 
 
+@pytest.mark.parametrize(
+    "response_body, message_id",
+    [
+        (
+            # Send single message to SQS response
+            b"{"
+            b'   "MD5OfMessageAttributes":"6e6aba56e93b3ddfdfe3fa28895feece",'  # pragma: allowlist secret
+            b'   "MD5OfMessageBody":"0d40eb1479f7e61b1a1c7a425c3949e4",'  # pragma: allowlist secret
+            b'   "MessageId":"c5aca29a-ff2f-4db5-94c3-90523d1ed4ca"'  # pragma: allowlist secret
+            b"}",
+            "c5aca29a-ff2f-4db5-94c3-90523d1ed4ca",
+        ),
+        (
+            # Send batch message request to SQS (one record in the batch), successful message
+            b"{"
+            b'   "Failed":[],'
+            b'   "Successful":['
+            b"     {"
+            b'        "Id":"1",'
+            b'        "MD5OfMessageBody":"68390233272823b7adf13a1db79b2cd7",'  # pragma: allowlist secret
+            b'        "MessageId":"c5aca29a-ff2f-4db5-94c3-90523d1ed4ca"'
+            b"     }"
+            b"]}",
+            "c5aca29a-ff2f-4db5-94c3-90523d1ed4ca",
+        ),
+        (
+            # Send batch message request to SQS (multiple record in the batch), successful message
+            # Note: Currently we only send the first message id of a batch, but if there would be a need we will change
+            #       this to send multiple message ids
+            b"{"
+            b'   "Failed":[],'
+            b'   "Successful":['
+            b"     {"
+            b'        "Id":"1",'
+            b'        "MD5OfMessageBody":"68390233272823b7adf13a1db79b2cd7",'  # pragma: allowlist secret
+            b'        "MessageId":"c5aca29a-ff2f-4db5-94c3-90523d1ed4ca"'
+            b"     },"
+            b"     {"
+            b'        "Id":"2",'
+            b'        "MD5OfMessageBody":"68390233272823b7adf13a1db7222222",'  # pragma: allowlist secret
+            b'        "MessageId":"c5aca29a-ff2f-4db5-94c3-90523d222222"'
```

Comment:
please change the string completely. this is super not readable that you change only the last 6 digits

Reply:
Done